content/guides/integrations/redis/Java.md

```diff
@@ -9,152 +9,167 @@ logo: /images/java-opencensus.png
 
-### Generate and install
-Inside the cloned repository's directory run
-```shell
-mvn install:install-file -Dfile=$(pwd)/target/jedis-3.0.0-SNAPSHOT.jar \
--DgroupId=redis.clients -DartifactId=jedis -Dversion=3.0.0 \
--Dpackaging=jar -DgeneratePom=true
-```
+### Imports
+One just needs to import the integration and use it like they would for Jedis
 
-## Enabling observability
-To enable observability, we'll need to use Jedis normally but with one change
+```java
+import io.orijtech.integrations.ocjedis.OcWrapJedis;
+import io.orijtech.integrations.ocjedis.Observability;
 
-{{<highlight java>}}
-import redis.clients.jedis.Observability;
-{{</highlight>}}
+private static final OcWrapJedis jedis = new OcWrapJedis(redisHost);
+```
 
-and then finally to enable metrics
+### Enabling observability
+We'll need to enable stats by registering and enabling all available views
 {{<highlight java>}}
 // Enable exporting of all the Jedis specific metrics and views
 Observability.registerAllViews();
 {{</highlight>}}
 
-## Available metrics
-Metric search suffix|Description
----|---
-redis/bytes_read|The number of bytes read from the Redis server
-redis/bytes_written|The number of bytes written out to the Redis server
-redis/dials|The number of connection dials made to the Redis server
-redis/dial_latency_milliseconds|The number of milliseconds spent performing Redis operations
-redis/errors|The number of errors encountered
-redis/connections_opened|The number of new connections
-redis/roundtrip_latency|The latency spent for various Redis operations
-redis/reads|The number of reads performed
-redis/writes|The number of writes performed
-
-## End to end example
+### Available metrics
+
+Metric search suffix|Description|Tags
+---|---|---
+jedis/latency|The latency of the various methods|"method", "error", "status"
+jedis/calls|The calls made by various methods|"method", "error", "status"
+jedis/data_transferred|The amount of data transferred|"method", "type"
+
+
+### End to end example
+
+In this end-to-end example, we've taken an excerpt from a search service. It checks Redis server if the expensively computed response
+was already saved. If not found ("cache miss"), it'll process the response then save it to the cache for later "cache hits".
+
 {{<tabs Source Pom>}}
 
 {{<highlight java>}}
 package io.opencensus.tutorials.jedis;
 
-import io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
-import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
-import io.opencensus.exporter.trace.stackdriver.StackdriverTraceConfiguration;
-import io.opencensus.exporter.trace.stackdriver.StackdriverTraceExporter;
+import io.opencensus.trace.Tracer;
 import io.opencensus.trace.Tracing;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.Status;
 import io.opencensus.trace.config.TraceConfig;
 import io.opencensus.trace.config.TraceParams;
```

Comment:
Unused import, please remove it.

Reply:
We use these two imports in the code, please take a look https://raw.githubusercontent.com/census-instrumentation/opencensus-website/b41039e38360fac7d6fc7d71a8cacdee09a89088/content/guides/integrations/redis/Java.md

Reply:
Thanks for the offline clarification that you meant TraceParams, I'll remove that.